lighthouse-core/report/html/templates.html

```diff
@@ -278,13 +286,10 @@
     .dark .lh-tools__button svg {
       filter: invert(1);
     }
-    .lh-tools__button:focus,
-    .lh-tools__button.active {
-      outline: none;
-    }
     .lh-tools__button.active + .lh-tools__dropdown {
       opacity: 1;
       clip: rect(-1px, 187px, 242px, -3px);
+      visibility: visible;
```

Comment:
anything extra this does for us compared to opacity?

Reply:
It removes the elements from focus consideration.